mne/preprocessing/xdawn.py

```diff
@@ -16,7 +16,7 @@
 from ..io.pick import _pick_data_channels
 from ..utils import logger
 from ..externals.six import iteritems, itervalues
-
+from ..stats.regression import _prepare_rerp_data, _prepare_rerp_preds, _solver
```

Comment:
It should be numerically identical to within, or at least very close to, machine precision for non-degenerate cases. Only when the matrix becomes very ill conditioned does the (non-regularized) cholesky perform a bit worse. On the other hand, it's very easy to regularize.

But yes, I wouldn't mind having both. Giving it a 'solver' kwarg param means it's easy to e.g. regularize.